cli/Cargo.toml

```diff
@@ -36,6 +36,7 @@ solana-cli-output = { workspace = true }
 solana-client = { workspace = true }
```

Comment:
It would be nice if we could remove this dependency from `solana-cli` altogether. It's the send-transaction stuff in program that's preventing that?